tests/Configuration/ConfigurationResponseTest.php

```diff
@@ -45,6 +47,41 @@ public function test_it_creates_a_configuration_response_with_minimal_data(): vo
         );
     }
 
+    public function test_it_creates_a_configuration_response_with_values(): void
+    {
+        $form = Mockery::mock(Form::class, [
+            'toArray'     => [],
+            'getRequired' => [],
+        ]);
+
+        $faker = Factory::create();
+
+        $name = $faker->word;
+        $value = $faker->word;
```

Comment:
faker will issue warning for usage of the magic props:
```suggestion
        $name = $faker->word();
        $value = $faker->word();
```

Reply:
Give warnings where? I don't see them in my IDE, nor terminal, nor pipeline

Reply:
ohhh, must be related to that flag then that I never noticed this before!